packages/client/src/components/GameSettings/GameSettings.tsx

```diff
@@ -0,0 +1,42 @@
+import { StFlex } from 'styles/global';
+import GameSettingsButton from './GameSettingsButton';
+
+interface GameSettingsType {
+  handleCloseModal: () => void;
+}
+
+const settingsInputsValue = [
+  {
+    value: 2,
+  },
+  {
+    value: 4,
+  },
+];
+
+const GameSettings = (props: GameSettingsType) => {
+  const { handleCloseModal } = props;
+
+  const chooseCountOfPlayers = (evt: { target: HTMLButtonElement }) => {
```

Comment:
Попробуй лучше 
```suggestion
  const chooseCountOfPlayers = (evt: React.SyntheticEvent<HTMLButtonElement>) => {
```
(код неточный, примерно такое было)